Dockerfile

```diff
@@ -1,10 +1,21 @@
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.0
+ARG APP_INSIGHTS_AGENT_VERSION=2.5.0-BETA.3
+
+# Build image
+
+FROM busybox as downloader
+
+RUN wget -P /tmp https://github.com/microsoft/ApplicationInsights-Java/releases/download/2.5.0-BETA.3/applicationinsights-agent-2.5.0-BETA.3.jar
+
+# Application image
+
+FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-debug-1.0
 
 EXPOSE 4000
 
 HEALTHCHECK --interval=10s --timeout=10s --retries=10 CMD http_proxy="" curl --silent --fail http://localhost:4000/health
```

Comment:
Do we want to remove health check at the same time? Something which was requested as part of the AKS.

Reply:
Separate PR.